src/apps/workspaces/domain/user_applet_access.py

```diff
@@ -39,3 +42,17 @@ class UserAppletAccessItem(InternalModel):
     user_id: uuid.UUID
     applet_id: uuid.UUID
     role: Role
+
+
+class RemoveRespondentAccess(InternalModel):
+    """Respondent access removal model."""
+
+    user_id: uuid.UUID = Field(
+        description="This field represents the applet respondent id",
+    )
+    applet_ids: list[uuid.UUID] = Field(
+        description="This field represents the applet ids",
+    )
+    delete_responses: bool = Field(
+        description="This field represents the flag for deleting responses",
+    )
```

Comment:
RemoveManagerAccess is quite the same as this model. Why they are in separate files? Could we simplify this a little?